packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh

```diff
@@ -5,5 +5,55 @@
 # CWD is agoric-sdk
 here=./upgrade-test-scripts/agoric-upgrade-11
```

Comment:
this precedes this PR, but I want to say I think we can have a better name for this. eg. `thisupgrade`. or find a way to evaluate the scripts letting them use file path relative references. maybe if we adopt zx or execa.